trajopt_utils/src/logging.cpp

```diff
@@ -18,7 +18,11 @@ int LoggingInit()
     std::printf("You can set logging level with TRAJOPT_LOG_THRESH. Valid values: "
                 "%s. Defaulting to INFO\n",
                 VALID_THRESH_VALUES);
-    lvlstr = "INFO";
+  #ifdef NDEBUG
```

Comment:
Why ERROR, DEBUG instead of DEBUG, INFO?

Reply:
I basically tried to tune down the verbosity by default, from the very verbose "INFO" to "ERROR". If you prefer to keep it more verbose I can change it back!

Reply:
Sounds good. If not set it will default to error but if built in debug it will be set to debug.

Reply:
Sound good.